lib/OctokitAction.ts

```diff
@@ -30,6 +34,22 @@ export abstract class OctokitAction extends Action {
     return this.graphqlWithAuth(query);
   }
 
+  protected async moveIssue(issueId: string, transitionName: string): Promise<void> {
+    try {
+      console.log(`${issueId}: Getting transition id for '${transitionName}'`);
```

Comment:
This logging should be inside the `listTransitions` method (see the other comment about layering)